TombLib/TombLib/Catalogs/TEN Node Catalogs/UI_Hud.lua

```diff
@@ -0,0 +1,563 @@
+--User interface Nodes by Adngel,DaviDMRR and TrainWreck ; Code references by Lwmte and TEN Nodes.
+local CustomBar = require("Engine.CustomBar")
+
+-- !Name "Create basic custom bar"
+-- !Section "User interface"
+-- !Conditional "False"
+-- !Description "Creates a bar with max value 1000"

```

Comment:
Descriptions should end with a full stop. Names should not. Also, please rephrase this, maybe like "Creates a bar with a maximum value of 1000".